di-ipv-cimit-stub/lambdas/stub-management/src/main/java/uk/gov/di/ipv/core/stubmanagement/StubManagementHandler.java

```diff
@@ -73,10 +87,12 @@ public APIGatewayProxyResponseEvent handleRequest(
                 String ci = pathParameters.get(CI_PATH_PARAMS);
                 UserMitigationRequest userMitigationRequest =
                         objectMapper.readValue(event.getBody(), UserMitigationRequest.class);
-                if (httpMethod.equals(HttpMethod.POST.toString())) {
-                    userService.addUserMitigation(userId, ci, userMitigationRequest);
-                } else if (httpMethod.equals(HttpMethod.PUT.toString())) {
-                    userService.updateUserMitigation(userId, ci, userMitigationRequest);
+                if (SUPPORTED_MITIGATION_METHODS.contains(httpMethod)) {
```

Comment:
I think the initial idea might have been to allow QA's to easily get stuff set up how they want it. I think it might have been one of those things that seems like a good idea, but in reality doesn't get used. 
We'd never use the PUT in a journey flow - it's more about manually tweaking stuff.